models/gpt2/gpt2.py

```diff
@@ -318,8 +322,14 @@ def _elish_fn(x):
         return _elish_fn
     
     # swish activations
-    elif activation_fn == "swish": # https://arxiv.org/abs/1710.05941
+    elif activation_fn == "silu": # https://arxiv.org/abs/1710.05941
         return mtf.swish
+    elif activation_fn == "swish":
+        def _swish_fn(x):
+            beta = mtf.get_variable(x.mesh, "beta", [], initializer=tf.constant_initializer(0), dtype=dtype)
```

Comment:
Just my personal opinion regarding styles, but it's possible to inline this into a single lambda function as you're only referring once to beta.
I think it'd also be important to add random names to the parameter, like "swish-beta-0012890", as it's entirely possible that this activation gets called multiple times within the same variable scope.
Also, should we perhaps use ARiA instead of swish as that's a more general description of exactly that function? 
Reference [implementation](https://github.com/NarendraPatwardhan/Aria/blob/master/ARiA.ipynb), fast.ai [discussion](https://forums.fast.ai/t/implementing-new-activation-functions-in-fastai-library/17697) and [paper](https://arxiv.org/abs/1805.08878)